src/RegistrationController.js

```diff
@@ -147,11 +151,16 @@ function (
         var name = schemaProperty.get('name');
         if (name === 'password' || name === 'login') {
           inputOptions.events = {
-            'input': function() {
+            'input': function () {
               checkPasswordMeetComplexities(this.model);
             }
           };
         }
+        if (name === 'password') {
+          inputOptions.events.focusout = function () {
```

Comment:
We tend to use `focusout` in the okta-signin-widget https://github.com/okta/okta-signin-widget/blob/096cc4cb219b6a6467730d24c4323effd240e85f/src/PrimaryAuthController.js .

Anyway, in backbone github they suggest to use `focusout` over `blur`, since, as HK said, the first is bubbled while the second no.